### ISSUE TITLE
jupyter labextension install @ryantam626/jupyterlab_black hangs

### ISSUE THREAD

**Issue by EyalSel** [ryantam626/jupyterlab_black#13]
Hi, I'm running the following command:
`jupyter labextension install @ryantam626/jupyterlab_black`

On a conda python2.7 environment after having run `pip install jupyterlab`.

Getting the following result:
```
(myenv_py27) root@fc865b1b006a:~/Metis12# jupyter labextension install @ryantam626/jupyterlab_black
> /root/anaconda2/envs/myenv_py27/bin/npm pack @ryantam626/jupyterlab_black
npm notice
npm notice package: @ryantam626/jupyterlab_black@0.2.3
npm notice === Tarball Contents ===
npm notice 1.2kB package.json
npm notice 879B  CHANGELOG.md
npm notice 1.1kB LICENSE
npm notice 3.3kB README.md
npm notice 231B  lib/index.d.ts
npm notice 4.0kB lib/index.js
npm notice 640B  schema/settings.json
npm notice 0     style/index.css
npm notice === Tarball Details ===
npm notice name:          @ryantam626/jupyterlab_black
npm notice version:       0.2.3
npm notice filename:      ryantam626-jupyterlab_black-0.2.3.tgz
npm notice package size:  4.5 kB
npm notice unpacked size: 11.4 kB
npm notice shasum:        bc722963db91ae0bde3a63032d116a599983343e
npm notice integrity:     sha512-oLL3s4mKROjH6[...]WRZG4lukw7m/A==
npm notice total files:   8
npm notice
ryantam626-jupyterlab_black-0.2.3.tgz
> node /root/anaconda2/envs/myenv_py27/lib/python2.7/site-packages/jupyterlab/staging/yarn.js install
yarn install v1.6.0
(node:8824) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
info No lockfile found.
[1/4] Resolving packages...
warning @jupyterlab/csvviewer-extension > @jupyterlab/apputils > react-dom > fbjs > core-js@1.2.7: core-js@<2.6.6 is no longer maintained. Please, upgrade to core-js@3 or at least to actual version of core-js@2.
⠁@ryantam626/jupyterlab_black@file:../extensions/ryantam626-jupyterlab_black-0.2.3.tgz
```

And that little dot on the left of the last line just keeps spinning - the installation never finishes.

Any ideas what might be happening?

Many thanks

**Comment by ryantam626** [comment]
Sorry not really sure why is this happening...

Maybe you try updating your node/yarn and retry?

**Comment by EyalSel** [comment]
I was able to resolve that issue following https://github.com/jupyterlab/jupyterlab/issues/5225.
The TL;DR is that I'm using JupyterLab version 0.33.12 (and not the newest version),  which is the highest available version for python2.7. Turns out this version of JupyterLab doesn't work with newer versions of Node, and I had to downgrade to node v.8 to get the installation to go through.

I'm now facing a different problem, however. While the installation seems to finish successfully and I'm able to enable the extension and then see it using `jupyter labextension list`, when I start jupyter lab I can't see the jupyter_black tab in the settings menu as shown in the README.

Any thoughts on what might be the issue?

Thanks again

**Comment by ryantam626** [comment]
Ah that's a good sign. Is there anything in the web broswer's console?
